src/main/java/uk/gov/hmcts/reform/prl/models/email/EmailTemplateNames.java

```diff
@@ -57,7 +57,9 @@ public enum EmailTemplateNames {
     FM5_REMINDER_APPLICANT_RESPONDENT,
     RESPONDENT_RESPONDED_CAFCASS,
     RESPONDENT_ALLEGATIONS_OF_HARM_CAFCASS,
-    RESPONDENT_RESPONDED_ALLEGATIONS_OF_HARM_CAFCASS
+    C7_NOTIFICATION_APPLICANT, C1A_NOTIFICATION_APPLICANT, RESPONDENT_RESPONDED_ALLEGATIONS_OF_HARM_CAFCASS,
```

Comment:
can we please follow same standard as having one template name per line

Reply:
Done @yogendraU Thank you !! Request please verify